app/gzac/src/main/resources/config/application.yml

```diff
@@ -45,7 +45,6 @@ spring:
         enabled: false
     jpa:
         database-platform: org.hibernate.dialect.MySQL57Dialect
-        database: mysql
```

Comment:
I'm guessing this is derived from database-platform instead?

Reply:
Volgens spring:

Target database to operate on, auto-detected by default. Can be alternatively set using the "databasePlatform" property.